chain/network/src/routing/routing_table_view.rs

```diff
@@ -37,12 +37,12 @@ pub struct RoutingTableView {
     /// If there are several options use route with minimum nonce.
     /// New routes are added with minimum nonce.
     route_nonce: LruCache<PeerId, usize>,
-    /// Ping received by nonce.
-    ping_info: LruCache<usize, (Ping, usize)>,
-    /// Ping received by nonce.
-    pong_info: LruCache<usize, (Pong, usize)>,
+    /// Pings received.
+    ping_info: Vec<Ping>,
+    /// Pongs received.
+    pong_info: Vec<Pong>,
     /// List of pings sent for which we haven't received any pong yet.
-    waiting_pong: LruCache<PeerId, LruCache<usize, Instant>>,
+    waiting_pong: LruCache<PeerId, LruCache<u64, Instant>>,
```

Comment:
I am forgetting whether ping/pongs can actually be reached in production? Might be worth adding a comment here about how we use ping/pongs

Reply:
agreed, I don't have yet a clear idea how to get rid of it.